data/outreachy.yml

```
@@ -275,5 +275,18 @@ rounds:
         mentors:
         - Reynir Björnsson
         video: https://watch.ocaml.org/w/kTwq4AVQtRtFuQx9R5cPro
+  - name: Summer 2023
+    projects:
+      - title: "Improving Error Reporting in Existing PPXLIB-Based PPXs"
+        description: >
+          In the past, when ppxlib encountered an exception in a transformation, it stopped the rewriting process, causing the rewriters after to not be processed, 
```

```suggestion
          In the past, when `ppxlib` encountered an exception in a transformation, it stopped the rewriting process, causing the rewriters after to not be processed, 
```

"...causing the rewriters that came after..." ?